src/systemctl/systemctl-set-environment.c

```diff
@@ -8,6 +8,44 @@
 #include "systemctl-util.h"
 #include "systemctl.h"
 
+static int json_transform_message(sd_bus_message *m, JsonVariant **ret) {
+        _cleanup_(json_variant_unrefp) JsonVariant *v = NULL, *w = NULL;
+        char *text, *sep;
```

Comment:
these can be moved inside the loop

Reply:
I've moved `sep` and `w`. I assume that's what you meant, right?